appveyor.yml

```diff
@@ -0,0 +1,91 @@
+version: 1.0-{build}
+image:
+- Visual Studio 2015
```

Comment:
MSVC 2015 libs are compatible with MSVC 2017 and 2019 (see https://docs.microsoft.com/en-us/cpp/porting/binary-compat-2015-2017?view=vs-2019)...  2019 is only at preview support in AppVeyor, but I feel like we should at least target 2017 if we can...

Reply:
I can bump it to VS2017 if desired.  It's been tested and does appear to work.  That is also the last version with support for the XP Toolset.